change/@ni-nimble-components-9135d7b9-acf1-411e-b8dc-8a73d7b3860f.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Banner Stories Severity Fix (#1350)",
+  "packageName": "@ni/nimble-components",
+  "email": "103057880+aidendk@users.noreply.github.com",
+  "dependentChangeType": "patch"
+}
```

This only affects docs and not the contents of the production package so it can use change type of `none`

```suggestion
{
  "type": "none",
  "comment": "Banner Stories Severity Fix (#1350)",
  "packageName": "@ni/nimble-components",
  "email": "103057880+aidendk@users.noreply.github.com",
  "dependentChangeType": "none"
}
```